### PR TITLE
Applied suggestions for Keylime

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: tasks/keylime-run-with-podman.xml [hunk 1]
```diff
@@ -76,54 +76,61 @@ registry.opensuse.org/devel/microos/containers/containerfile/opensuse/keylime-co
...
   </section>
   <section xml:id="keylime-extract-certificates">
-    <title>Extracting &keylime; certificate</title>
+    <title>Extracting the &keylime; certificate</title>
     <para>
       The first time that the &keylime; container is executed, its services
-      create a certificate required by several agents. You can extract the
-      certificate using the following command:
+      create a certificate required by several agents. You need to extract the
+      certificate from the container and copy it the agent's
```
Comment (lvicoun) [inline tasks/keylime-run-with-podman.xml:121]:
```suggestion
      certificate from the container and copy it to the agent's
```

File: tasks/keylime-run-with-podman.xml [hunk 2]
```diff
@@ -76,54 +76,61 @@ registry.opensuse.org/devel/microos/containers/containerfile/opensuse/keylime-co
       <para>
         The <literal>keylime-control-plane-container</literal> container is
         created. It includes configured and running registrar and verifier
-        services. Internally the container exposes several ports into the host
-        using the default values. Validate the firewall configuration to allow
-        access to the ports and to allow communication between containers,
-        because the tenant CLI requires it.
+        services. Internally the container exposes ports 8881, 8890 and 8891 to
```
Comment (dariavladykina) [inline tasks/keylime-run-with-podman.xml:79]:
```suggestion
        services. Internally, the container exposes ports 8881, 8890 and 8891 to
```

File: tasks/keylime-run-with-podman.xml [hunk 3]
```diff
@@ -76,54 +76,61 @@ registry.opensuse.org/devel/microos/containers/containerfile/opensuse/keylime-co
...
+      To view the logs of &keylime; services, run the following command:
     </para>
-<screen>podman ps</screen>
-<screen>podman logs keylime-control-plane-container</screen>
+<screen>&prompt.root;podman logs keylime-control-plane-container</screen>
   </section>
   <section xml:id="keylime-executing-tenant">
     <title>Executing the tenant CLI</title>
     <para>
       The tenant CLI tool is included in the container and if the host firewall
```
Comment (dariavladykina) [inline tasks/keylime-run-with-podman.xml:107]:
```suggestion
      The tenant CLI tool is included in the container, and if the host firewall
```

**Review by dariavladykina — COMMENTED** [review]
Hi, just minor things here. Thanks!

**Pull request merged**